src/pybind/mgr/dashboard/frontend/package.json

```diff
@@ -118,13 +119,16 @@
     "@types/simplebar": "5.1.1",
     "codelyzer": "6.0.0",
     "cypress": "4.12.1",
+    "cypress-multi-reporters": "1.4.0",
     "html-linter": "1.1.1",
     "htmllint-cli": "0.0.7",
     "identity-obj-proxy": "3.0.0",
     "jest": "26.3.0",
     "jest-canvas-mock": "2.2.0",
     "jest-preset-angular": "8.2.1",
     "jest-silent-reporter": "0.2.1",
+    "mocha": "^8.1.2",
```

Comment:
```suggestion
    "mocha": "8.1.2",
```

Reply:
Thanks! My bad. I initially added those by hand and then did it with `npm i --save-dev`...